linkerd-viz/values.yaml

```diff
@@ -1,5 +1,16 @@
-linkerd-viz: {}
-  # grafana:
-  #   enabled: false
-  # prometheus:
-  #   enabled: false
\ No newline at end of file
+linkerd-viz:
```

Comment:
Should we apply the "values-common.yaml, values-dev.yaml, values-live.yaml" structure to linkerd-viz deployment as well? @ldb

Reply:
Resolved 👍